CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.2.25]
+
+### Feat
+
+- Snyk Code: Adds setting a path for a custom LS binary
```

Comment:
This isn't "Snyk Code"specific configuration.

Reply:
I see, is there a preferred prefix here? `Snyk LSP: `? Looks like we prefix our tests with the specific product/feature. Or is no prefix fine here too

Reply:
Let's do `Snyk LS:`. Like the idea 👍